src/server/auth.ts

```diff
@@ -74,3 +75,50 @@ export const getServerAuthSession = (ctx: {
 }) => {
   return getServerSession(ctx.req, ctx.res, authOptions);
 };
+
+
+/*
+todo write docs
+*/
+export interface session {
+  account : Number;
+  token : String;
+  expiry : Date;
+}
+
+export const validateUser = async (
+  id:number, 
+  token:string
+  ) => {
+
+    const result = await prisma.session
+    .findMany({where: 
+            { 
+              account: id,
+              token : token 
+            } } )
+    .catch(message => {
+              throw new TRPCError({
```

Comment:
```suggestion
              console.error(`Something went wrong when trying to find account ${id} and token ${token}.`);
              throw new TRPCError({
```

I think being detailed with error messages will help us in the long term.